src/components/learner-credit-management/BudgetDetailPage.jsx

```diff
@@ -44,18 +57,32 @@ const BudgetDetailPage = ({ enterpriseSlug }) => {
             />
           </Col>
         </Row>
-        <BudgetDetailTabsAndRoutes />
```

Comment:
I believe we'll want to keep `BudgetDetailTabsAndRoutes` in place here; otherwise we'd be removing the tabs from the page to only render the "Activity" tab's contents.